cmd/minishift/cmd/config/view_test.go

```diff
@@ -74,6 +77,53 @@ func TestConfigView(t *testing.T) {
 
 		template := determineTemplate(tt.template)
 		configView(configTest, template, tee.StdoutBuffer)
-		assert.Equal(t, tt.expectedString, tee.StdoutBuffer.String())
+		var b bytes.Buffer
+		writer := bufio.NewWriter(&b)
+		formatAndWriteLines(writer, strings.Split(tt.expectedString, "\n"))
+		expectedString := b.String()
+		assert.Equal(t, expectedString, tee.StdoutBuffer.String())
+	}
+}
+
+func TestFormatAndWriteLines(t *testing.T) {
+	type args struct {
+		lines []string
+	}
+	tests := []struct {
+		args       args
+		wantWriter string
+	}{
+		{
+			args: args{
+				lines: []string{
+					"small: SomeValue",
+					"veryLongKeyForTestingAgainstSmallKey: AnotherValue",
+				},
+			},
+			wantWriter: "small\t\t\t\t\t: SomeValue\nveryLongKeyForTestingAgainstSmallKey\t: AnotherValue\n",
+		},
+		{
+			args: args{
+				lines: []string{
+					"smallKeyOne: SomeValue",
+					"smallKeyTwo: AnotherValue",
+				},
+			},
+			wantWriter: "smallKeyOne\t: SomeValue\nsmallKeyTwo\t: AnotherValue\n",
+		},
+		{
+			args: args{
+				lines: []string{
+					"urlKey: 'http://minishift.io/'",
+				},
+			},
+			wantWriter: "urlKey\t: 'http://minishift.io/'\n",
+		},
+	}
+	for _, tt := range tests {
+		writer := &bytes.Buffer{}
+		formatAndWriteLines(writer, tt.args.lines)
+		gotWriter := writer.String()
+		assert.Equal(t, tt.wantWriter, gotWriter)
```

Comment:
`s/wantWriter/expected`
`s/gotWriter/actual`

No need to state the implementation type...

Reply:
Have updated the naming